lib/web_ui/test/dom_renderer_test.dart

```diff
@@ -8,7 +8,17 @@ import 'dart:html' as html;
 import 'package:ui/src/engine.dart';
 import 'package:test/test.dart';
 
+import 'mobile_unit_test_bindings.dart';
+
 void main() {
+  MobileUnitTestBindings mobileUnitTestBindings =
+      isIosSafari ? MobileUnitTestBindings() : null;
+  setUpAll(() {
+    if (isIosSafari) {
+      mobileUnitTestBindings.initialize();
+    }
+  });
```

Comment:
I imagine the setup/tear down code will be identical across all tests. What do you think of moving it into a utility function? So in tests we only need to do this:

```dart
void main() {
  enableMobileSupport();

  // the rest of the code is the same
}
```